components/Storyblok/SbStoryMvp.tsx

```diff
@@ -0,0 +1,136 @@
+import { storyblokEditable, type SbBlokData } from '@storyblok/react/rsc';
+import Image from 'next/image';
+import { CreateBloks } from '@/components/CreateBloks';
+import { Masthead } from '@/components/Masthead';
+import { StoryHeroMvp, type StoryHeroMvpProps } from '@/components/Hero';
+import { ScrollyTelling } from '@/components/ScrollyTelling/ScrollyTelling';
+import { ScrollyFullwidth } from '../ScrollyTelling/ScrollyFullwidth';
+import { ScrollyDataViz } from '../ScrollyTelling/ScrollyDataViz';
+import { ProgressStory } from '../StoryPoC/ProgressStory';
+import { MulticolumnStory } from '../StoryPoC/MulticolumnStory';
+import { ChatbotStory } from '../StoryPoC/ChatbotStory';
+import { VideoScrollStory } from '../StoryPoC/VideoScrollStory';
+import { Bookshelf } from '../Bookshelf/Bookshelf';
+import { BrochureStory } from '../StoryPoC/BrochureStory';
+import { getNumBloks } from '@/utilities/getNumBloks';
+import { getProcessedImage } from '@/utilities/getProcessedImage';
+
+type SbStoryMvpProps = {
+  blok: {
+    _uid: string;
+    content?: SbBlokData[];
+    ankle?: SbBlokData[];
+  } & StoryHeroMvpProps;
+  slug?: string;
+};
+
+export const SbStoryMvp = ({
```

Comment:
Create a different Story MVP component/page type, and kept the original Story page type, since they still like the old style and might go to that in the future.